templates/marathon.tmpl

```diff
@@ -1,13 +1,13 @@
 {{$apps := .Applications}}
 [backends]{{range .Tasks}}
-    [backends.backend{{.AppID | replace "/" "-"}}.servers.server-{{.ID | replace "." "-"}}]
+    [backends.backend{{with index $apps 0 }}{{getBackend .}}{{end}}.servers.server-{{.ID | replace "." "-"}}]
```

Comment:
I think there is an issue here `with index $apps 0` :'(
You should have send all apps, like in `getProtocol` and compare the ID in the fonction I think.